input/postgres/relation_stats_ext.go

```diff
@@ -0,0 +1,105 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/lib/pq"
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+)
+
+const extendedStatisticsSQLExprsField = "ARRAY[]::text[]"
+const extendedStatisticsSQLpg14ExprsField = "se ->> 'exprs'"
+const extendedStatisticsSQLInheritedField = "null"
+const extendedStatisticsSQLpg15InheritedField = "se ->> 'inherited'"
+
+const extendedStatisticsSQL = `
+WITH relation_stats_ext AS (
+	SELECT se ->> 'statistics_schemaname' statistics_schemaname,
+		   se ->> 'statistics_name' statistics_name,
+		   %s exprs,
+		   %s inherited,
+		   se ->> 'n_distinct' n_distinct,
+		   se ->> 'dependencies' dependencies
+	  FROM %s
+)
+SELECT c.oid,
+	   n.nspname,
+	   s.stxname,
+	   (SELECT array_agg(k) FROM unnest(s.stxkeys) k) stxkeys,
+	   sd.exprs,
```

Comment:
As discussed, we should change that back to `pg_get_statisticsobjdef_expressions(pg_stats_ext.exprs)` to not require extra permissions for getting the definition of the statistics object.